platform-includes/feature-flags/change-tracking-cta-launchdarkly/_default.mdx

```diff
@@ -0,0 +1,9 @@
+## Change Tracking
+
+Sentry can track changes to feature flag definitions and report suspicious feature flag edits. To enable the change tracking integration click the "Set Up Integration" button on the feature flag table contained within the Issue Details page. This will trigger a fly out which will guide you through the two step process.
+
```

Comment:
```suggestion
Sentry can track changes to feature flag definitions and report suspicious feature flag edits. To enable the change-tracking integration, click the "Set Up Integration" button on the feature flag table, which you'll find on the **Issue Details** page. This will trigger a flyout that will guide you through the two-step process.

You can navigate to the **Issue Details** page by clicking any issue on the [**Issues**](https://sentry.io/orgredirect/organizations/:orgslug/issues/) page.

```